src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/config.jelly

```diff
@@ -54,6 +54,14 @@
         <f:textbox default="32"/>
     </f:entry>
 
+    <f:entry title="Max pod reconnection attempts" field="maxPodReconnectionAttempts">
+        <f:textbox default="100"/>
```

Comment:
you can use `f:number` https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#form:number